### ISSUE TITLE
Run command when alt detected

### ISSUE THREAD

**Issue by chelminski** [Bobcat00/AltDetector#2]
As written in the topic. Allow the command to be executed when the plugin detects multi-accounts. This will allow you to customize additional actions.

The plugin is installed in the lobby and currently the administration does not receive proactive information. I can configure everything perfectly, but I need the plugin to execute the command when it detects a user.

**Comment by Bobcat00** [comment]
I'm not sure if this is easy to do, because that part of the code is just getting formatted strings. But what parameters/placeholders would you suggest be available in the command? For example, what would the command you want look like?

**Comment by chelminski** [comment]
# Example commands:
_(Executed when multi-player accounts are found)_

###  Many commands can be opened logically using the placeholder: [Change-Output Placeholder](https://github.com/Kqliber/Expansion-ChangeOutput)
For example: If the number of multi-accounts is greater than X run command: ban ip <player ip>
For this you will need placeholders listed in the next chapter.


###  Discord, Slack, Email notifications, etc.
Commands sent to trigger the creation of flow data to an external portal. For example, posting on the Administration channel that the player has more than 3 accounts. You can use ChangeOutput for this too.
For example: `Send a notification to the administration via Discord if the player has more than 1 account.`
For example: `Send an email to the player reminding them of the rules that they cannot have more than one account.`

# Needed placeholders:

- ~~Player name~~ - PlaceholderApi
- ~~Player IP~~ - PlaceholderApi

### To do:

- List of multi-account nicknames (no comma, only space) | [user1 user2 user3 user4]
- Number of multi-accounts | [4]
- Does the player have multi-accounts? [true/false]

# Conclusion

**To sum up, we need to add placeholders only.**
To execute the command when a player joins the server we can use ["JoinCommands Plugin"](https://github.com/SirBlobman/JoinCommands)
So I'm changing my request a bit. We only need placeholders. The commands will be executed automatically after the player joins the server.

**Comment by Bobcat00** [comment]
I'll consider this, but let me say that banning players based on this is a bad idea. Here are a bunch of cases where legitimate players share IP addresses:

- Siblings or other relatives living in the same house
- People bringing their laptops to a friend's house to play
- People using CGNAT, such as Starlink or cell phone hotspot customers
- People who get new IP addresses often (some ISPs change them daily) and happen to get one you've already seen

Do you really want to ban players just because they have friends or use Starlink?

**Comment by chelminski** [comment]
Yes. For example, if there are more than 5 accounts from a given IP address, they should be banned. My plan assumes that the administration should receive notifications when someone has 3+ accounts. If someone creates accounts faster than my administration is able to contact the player asking for an explanation - I think such accounts should be banned. I was already using the multi-account verification system, but it stopped working and had no support. I realize that many people can legally have the same IP and that's nothing bad. :)

**Comment by Bobcat00** [comment]
I made a beta version of AltDetector that supports PluginAPI expansions (and Discord webhooks).
Placeholder support is enabled by default. The format is:
`%altdetector_alts_<playername>%`
\<playername\> is case-sensitive
Download it here:
https://github.com/Bobcat00/AltDetector/releases/tag/v2.04-beta1
